administrator/components/com_cookiemanager/src/Table/GroupTable.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_cookiemanager
+ *
+ * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\Cookiemanager\Administrator\Table;
+
+\defined('_JEXEC') or die;
+
+use Joomla\CMS\Application\ApplicationHelper;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\String\PunycodeHelper;
+use Joomla\CMS\Table\Table;
+use Joomla\CMS\Tag\TaggableTableInterface;
+use Joomla\CMS\Tag\TaggableTableTrait;
+use Joomla\CMS\Versioning\VersionableTableInterface;
+use Joomla\Database\DatabaseDriver;
+use Joomla\String\StringHelper;
+use Joomla\Registry\Registry;
+
+/**
+ * Group Table class.
+ *
+ * @since   __DEPLOY_VERSION__
+ */
+class GroupTable extends Table implements VersionableTableInterface, TaggableTableInterface
+{
```

Comment:
Dont implement Interfaces you dont't use in your component - you want to extend only Table.